tensorboard/webapp/routes/dashboard_deeplink_provider_test.ts

```diff
@@ -310,6 +310,15 @@ describe('core deeplink provider', () => {
           expect(state.metrics.pinnedCards).toEqual(expectedPinnedCards);
         }
       });
+
+      it('associated unrecognized query params with other plugins', () => {
```

Comment:
I think it would be good to either expand this test or add another one that ensures that things like "tag filter" don't get stored in pluginQueryParams.